src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -564,8 +564,6 @@ private PollingResult compareRemoteRevisionWithImpl(Job<?, ?> project, Launcher
 
         final EnvVars pollEnv = project instanceof AbstractProject ? GitUtils.getPollEnvironment((AbstractProject) project, workspace, launcher, listener, false) : lastBuild.getEnvironment(listener);
 
-        final String singleBranch = getSingleBranch(pollEnv);
-
```

Comment:
move to actual usage. Considering this fix I've been confused to see this variable computed before the remote polling code being ran, wondered there was some unexpected use